packages/core-flows/src/common/steps/remove-remote-links.ts

```diff
@@ -35,7 +35,7 @@ export const removeRemoteLinkStep = createStep(
     )
     await link.delete(grouped)
 
-    return new StepResponse(void 0, grouped)
```

Comment:
There is a typing issue where if you return `void 0` you cannot do `.config` on the step to change the name